indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -40,10 +44,18 @@
                     disabled: false,
                     message: $T.gettext("Not possible for categories containing events")
                 },
+                categoriesDescendingFrom: {
+                    disabled: false,
+                    ids: [],
+                    // Closest parent category title will be used if more than one parent is present in ids
```

Comment:
Since you go for a sentence with a proper syntax, why don't you add `The` in the front?